packages/lib-content/package.json

```diff
@@ -49,6 +49,7 @@
     "styled-components": "5.x.x"
   },
   "devDependencies": {
+    "@babel/cli": "^7.25.9",
```

Comment:
Had to install `@babel/cli` in a couple of our libraries because they use the `babel` command in the `build` scripts.